modules/wallet/outputs.go

```diff
@@ -39,7 +39,7 @@ type key struct {
 func (w *Wallet) findOutputs(amount consensus.Currency) (knownOutputs []*knownOutput, total consensus.Currency, err error) {
 	w.update()
 
-	if amount == consensus.Currency(0) {
+	if amount.Sign() == 0 {
 		err = errors.New("cannot fund 0 coins") // should this be an error or nil?
```

Comment:
wrt the comment, it should probably stay an error. We can probably delete this comment.